lapis2-docs/astro.config.mjs

```diff
@@ -35,8 +35,8 @@ export default defineConfig({
                     label: 'References',
                     items: [
                         {
-                            label: 'Endpoints',
-                            link: '/references/endpoints/',
+                            label: 'Introduction',
```

Comment:
This clashes with the other page that is already named "Introduction" (i.e. the link is not unique enough for the e2e tests to click it). Should we name it a bit more specifically for this section for disambiguation?